http/src/test/resources/reference.conf

```diff
@@ -300,7 +300,7 @@ azure {
    pubsub-handler {
         create-vm-poll-config {
            initial-delay = 1 seconds
-           max-attempts = 120
+           max-attempts = 10
            interval = 1 seconds
```

Comment:
Do we want to increase the interval to give the service more time to come up?

Reply:
nah...this is only for unit tests where we just mock the results typically..so shorter attempts make sense